openpype/hosts/maya/plugins/create/create_render.py

```diff
@@ -71,7 +71,7 @@ class CreateRender(plugin.Creator):
     label = "Render"
     family = "rendering"
     icon = "eye"
-
+    enable_all_lights = True
```

Comment:
Is this attribute unused?

Reply:
```suggestion
```
I suspect this attribute is unused and can be removed.

Reply:
Ah, good catch, thank you Roy